README.md

```diff
@@ -11,10 +11,13 @@ Run `docker-compose up --build`.
 `npm i --force`
 
 ### Run
+**IMOPRTANT!!! To change collections prefix copy `<root>/.env` file to `<root>/ui` with proper env variable.**
```

Comment:
Be here more precise: 
... with proper `NG_COLLECTIONS_PREFIX` value. 
Did you mean that here?
You can also give an example of such value

Reply:
done